ArmaforcesMissionBot.Tests/Features/Signups/Importer/SignupImporterTests.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ArmaforcesMissionBot.Exceptions;
+using ArmaforcesMissionBot.Features;
+using ArmaforcesMissionBot.Features.Signups.Importer;
+using Discord.Commands;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace ArmaforcesMissionBot.Tests.Features.Signups.Importer {
+    public class SignupImporterTests {
+        [Theory]
+        [InlineData("Ddd\ndsd", "\n")]
+        [InlineData("Ddd\r\ndsd", "\r\n")]
+        [InlineData("Ddddsd", "\n")]
+        public void GetLineEndings(string message, string expectedLinebreak) {
+            var lineEnding = SignupImporter.GetLineEnding(message);
+
+            lineEnding.Should().Be(expectedLinebreak);
+        }
+
+        [Theory]
+        [InlineData("Ddd\ndsds\n", new[] {"Ddd\n", "dsds\n", "\n"})]
+        [InlineData("Ddd\r\ndsds\r\n", new[] {"Ddd\r\n", "dsds\r\n", "\r\n"})]
+        [InlineData("", new[] {"\n"})]
+        public void ReadLines(string message, string[] expectedLines) {
+            var linesRead = SignupImporter.ReadLines(message);
+
+            linesRead.Should().BeEquivalentTo(expectedLines);
+        }
+
+        [Theory]
+        [InlineData(new[] {"dsdsd", "AF!hsda", "#dsds", "asdas"}, new[] { "hsda\nasdas" })]
+        [InlineData(new[] {"AF!dsdsd", "AF!hsda", "#dsds", "asdas"}, new[] { "dsdsd", "hsda\nasdas" })]
+        [InlineData(new[] {"AF!dsdsd", "hsda", "#AF!dsds", "asdas"}, new[] { "dsdsd\nhsda\nasdas" })]
+        [InlineData(new[] {"AF!dsdsd", "", "asdas"}, new[] { "dsdsd\n\nasdas" })]
+        [InlineData(new[] {"dsdsd", "hsda", "#AF!dsds", "asdas"}, new string[0])]
+        [InlineData(new[] {"dsdsd", "hsda", "//AF!dsds", "asdas"}, new string[0])]
+        public void ParseCommands(IEnumerable<string> lines, IEnumerable<string> expectedCommandsList) {
```

Comment:
I'd include also a test for AF! moniker in the middle of a word, eg. "dddAF!dupa"